python/ray/serve/controller.py

```diff
@@ -477,22 +480,21 @@ def list_deployments(self, include_deleted: Optional[bool] = False) -> bytes:
             )
         return deployment_route_list.SerializeToString()
 
-    def get_deployment_statuses(self) -> bytes:
-        """Gets the current status information about all deployments.
+    def get_serve_status(self) -> bytes:
 
-        Returns:
-            DeploymentStatusInfoList's protobuf serialized bytes
-        """
-        from ray.serve.generated.serve_pb2 import DeploymentStatusInfoList
-
-        deployment_status_info_list = DeploymentStatusInfoList()
-        for (
-            name,
-            deployment_status_info,
-        ) in self.deployment_state_manager.get_deployment_statuses().items():
-            deployment_status_info_proto = deployment_status_info.to_proto()
-            deployment_status_info_proto.name = name
-            deployment_status_info_list.deployment_status_infos.append(
-                deployment_status_info_proto
-            )
-        return deployment_status_info_list.SerializeToString()
+        # TODO (shrekris-anyscale): Replace defaults with actual REST API status
```

Comment:
can you explain?

Reply:
I hardcoded some defaults into `get_serve_status()`. Since the task deployment change hasn't been finished, there's no info to populate the `app_status` with yet.

Once the task deployment change is merged in, we can return the actual `app_status` instead of hardcoded defaults.

Reply:
sgtm